lib/Support/Unix/Path.inc

```diff
@@ -199,6 +199,8 @@ std::string getMainExecutable(const char *argv0, void *MainAddr) {
   char link_path[MAXPATHLEN];
   if (realpath(DLInfo.dli_fname, link_path))
     return link_path;
+#elif defined(__native_client__)
+  return "";
```

Comment:
is there any consequence to not being able to return the path to the executable?

Reply:
`getMainExecutable` is used to print stack traces and find other llvm tools, so there's no impact in this case.